program.py

```diff
@@ -0,0 +1,4 @@
+def div(a,b):
+    return a/b
```

Comment:
I think that you didn't consider the case when b is equal to 0. Could you add a check for that please?

Reply:
Sure I've forgot about it.I will update there pr in a sec.Thanks

Reply:
- [ ] Done